python-checks/src/test/resources/checks/identityComparisonWithCachedTypes.py

```diff
@@ -156,3 +156,7 @@ def comparison_to_none():
 def potential_null_symbols():
     type(arr) is tuple
     some_thing is other_thing
+
+def unknown_is_not_type(arg):
```

Comment:
Does that method name make sense?
I guess the underlying problem is to not confuse a type object with an object of a given type?

Reply:
The name does not make sense. You're absolutely right with the reason why the FP is happening.